### DISCUSSION TITLE
Student Development Application Success or Failure?

### DISCUSSION THREAD

**Discussion by mxt1022** [community/community#114672] [GitHub Education] [answered]
### Hi everyone,

I received the Academic Accreditation Failed email after my application was approved, but when I returned to the application screen the following image appeared:
![image](https://github.com/community/community/assets/88814303/b34b85bf-a476-4ff2-bce3-ea71a80e583a)
![image](https://github.com/community/community/assets/88814303/98aca8e0-30cc-4c97-90f7-06e7c4ac0f18)
![image](https://github.com/community/community/assets/88814303/1717bbae-75f6-4b28-ba0c-554b37eb3758)

Was my application successful or unsuccessful?

**Comment by ethanwater** [comment, marked as answer]
You have a PRO account now. So if you didn't pay for it, then yeah- It worked.

**Reply by mxt1022** [reply]
Okay, thank you very much.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
